FirebaseABTesting/Sources/FIRExperimentController.m

```diff
@@ -191,6 +186,21 @@ - (void)updateExperimentsWithServiceOrigin:(NSString *)origin
   });
 }
 
+- (void)updateExperimentsWithServiceOrigin:(NSString *)origin
+                             lastStartTime:(NSTimeInterval)lastStartTime
+                                  payloads:(NSArray<NSData *> *)payloads {
+  FIRLifecycleEvents *lifecycleEvent = [[FIRLifecycleEvents alloc] init];
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+  [self updateExperimentsWithServiceOrigin:origin
+                                    events:lifecycleEvent
+                                    policy:ABTExperimentPayloadExperimentOverflowPolicyDiscardOldest
```

Comment:
Will this always be the case for interop, or do we want it to be a parameter as well?

Reply:
Perhaps we want it to be the default for now? If we need to support other policies, we could add an API and expose the policies via the interop as well.

Reply:
SGTM, thanks